src/submit_referendum.rs

```diff
@@ -648,7 +714,18 @@ fn handle_batch_of_calls(output: &Output, batch: Vec<NetworkRuntimeCall>) {
 fn print_output(output: &Output, network_call: &NetworkRuntimeCall) {
 	match network_call {
 		NetworkRuntimeCall::Kusama(call) => {
-			let rpc: &'static str = "wss%3A%2F%2Fkusama-rpc.dwellir.com";
+			let rpc: &'static str = "wss%3A%2F%2Fkusama-rpc.n.dwellir.com";
+			match output {
+				Output::CallData => println!("0x{}", hex::encode(call.encode())),
+				Output::AppsUiLink => println!(
+					"https://polkadot.js.org/apps/?rpc={}#/extrinsics/decode/0x{}",
```

Comment:
Maybe one day we change these to a PAPI link

Reply:
Today's the day: [5ef7a15](https://github.com/joepetrowski/opengov-cli/pull/53/commits/5ef7a15cb0d1c97c20e57b31b7de7709264b73bf)